pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1870,10 +1872,14 @@ protected void handleAck(CommandAck ack) {
             Consumer consumer = consumerFuture.getNow(null);
             Subscription subscription = consumer.getSubscription();
             if (subscription.getTopic().isTransferring()) {
+                PulsarService pulsar = getBrokerService().getPulsar();
                 // Message acks are silently ignored during topic transfer.
+                long ignoredAckCount = ack.getMessageIdsCount();
+                var ignoredAckTotalCount = ExtensibleLoadManagerImpl.get(pulsar).getIgnoredAckCount().
```

Comment:
Should we check if the `ExtensibleLoadManagerImpl` is enabled?

Reply:
> the topic transferring flag can only be set when the Extensible Load Manager is enabled.

yes, since this transferring flag is set by ExtensibleLoadManagerImpl.  checking if ExtensibleLoadManagerImpl is enabled here seems redundant,